extensions/hibernate-search-orm-elasticsearch/deployment/src/main/resources/dev-templates/embedded.html

```diff
@@ -1,4 +1,4 @@
 <a href="{urlbase}/entity-types" class="badge badge-light">
     <i class="fa fa-search fa-fw"></i>
-    Indexed entity types <span class="badge badge-light">{info:indexedEntityTypes.size()}</span></a>
```

Comment:
You may want to rename `indexedEntityTypes` to `persistenceUnits`.

As it is, this code is a bit confusing :)

Also, you could display the number of persistence units *and* the total number of indexed entities, similarly to [what's done for the Hibernate ORM card](https://github.com/quarkusio/quarkus/blob/747b9ac22199633973d0d671146b5cad45d370c7/extensions/hibernate-orm/deployment/src/main/resources/dev-templates/embedded.html#L1-L7).

Reply:
done